libenkf/applications/ert_tui/main.c

```diff
@@ -78,7 +78,7 @@ void enkf_usage() {
   printf(" **                            E R T                                **\n");
   printf(" **                                                                 **\n");
   printf(" **-----------------------------------------------------------------**\n");
-  printf(" ** You have sucessfully started the ert program developed at       **\n");
+  printf(" ** You have successfully started the ert program developed at       **\n");
```

Comment:
Do not try and align the stars, that's impossible.

Instead, only try to realize the truth … there are no stars.